kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/immersivepetroleum/coker.js

```diff
@@ -0,0 +1,26 @@
+onEvent('recipes', (event) => {
+    if (global.isExpertMode == false) {
+        return;
+    }
+    const id_prefix = 'enigmatica:expert/immersivepetroleum/coker/';
+
+    recipes = [
```

Comment:
missing `const` :o

Reply:
Noes, `recipes` has no `var`, `let` or `const`, unless recipes is defined somewhere else this'll fail